Assets/Mirage/Runtime/NetworkServer.cs

```diff
@@ -193,13 +193,12 @@ public void StartServer(NetworkClient localClient = null)
             logger.Assert(Players.Count == 0, "Player could should have been reset since previous session");
             logger.Assert(connections.Count == 0, "connections could should have been reset since previous session");
 
-            LocalClient = localClient;
-            MessageHandler = new MessageHandler(DisconnectOnException);
-            MessageHandler.RegisterHandler<NetworkPingMessage>(Time.OnServerPing);
-
             World = new NetworkWorld();
             SyncVarSender = new SyncVarSender();
 
+            LocalClient = localClient;
+            MessageHandler = new MessageHandler(DisconnectOnException);
+            MessageHandler.RegisterHandler<NetworkPingMessage>(World.Time.OnServerPing);
```

Comment:
```suggestion
            MessageHandler.RegisterHandler<NetworkPingMessage>(Time.OnServerPing);
```

I assume this is a mis-copy from a future change?

Reply:
opps